src/GTranslate/LanguageDictionary.cs

```diff
@@ -142,153 +142,167 @@ public bool TryGetLanguage(string code, [MaybeNullWhen(false)] out Language lang
         ["ceb"] = new("Cebuano", "Binisaya", "ceb", "ceb", TranslationServices.Google | TranslationServices.Yandex),
-        ["ny"] = new("Chichewa", "Nyanja", "ny", "nya", TranslationServices.Google),
-        ["zh-CN"] = new("Chinese (Simplified)", "中文 (简体)", "zh-CN", "zho-CN"),
-        ["zh-TW"] = new("Chinese (Traditional)", "繁體中文 (繁體)", "zh-TW", "zho-TW", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
         ["co"] = new("Corsican", "Corsu", "co", "cos", TranslationServices.Google),
-        ["hr"] = new("Croatian", "Hrvatski", "hr", "hrv"),
         ["cs"] = new("Czech", "Čeština", "cs", "ces"),
+        ["cv"] = new("Chuvash", "Чӑвашла", "cv", "chv", TranslationServices.Yandex),
+        ["cy"] = new("Welsh", "Cymraeg", "cy", "cym"),
         ["da"] = new("Danish", "Dansk", "da", "dan"),
-        ["nl"] = new("Dutch", "Nederlands", "nl", "nld"),
+        ["de"] = new("German", "Deutsch", "de", "deu"),
+        ["dgo"] = new("Dogri", "डोगरी (ḍogrī) ڈوگرى", "dgo", "dgo", TranslationServices.Google),
+        ["dv"] = new("Divehi", "ދިވެހިބަސް", "dv", "div", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["ee"] = new("Ewe", "Eʋegbe", "ee", "ewe", TranslationServices.Google),
+        ["el"] = new("Greek", "Ελληνικά", "el", "ell"),
+        ["emj"] = new("Emoji", "Emoji", "emj", "emj", TranslationServices.Yandex), // Not present in Yandex.Cloud
         ["en"] = new("English", "English", "en", "eng"),
+        ["en-GB"] = new("English (UK)", "English (UK)", "en-GB", "en-GB", TranslationServices.Google),
         ["eo"] = new("Esperanto", "Esperanto", "eo", "epo", TranslationServices.Google | TranslationServices.Yandex),
+        ["es"] = new("Spanish", "Español", "es", "spa"),
+        ["es-MX"] = new("Spanish (Mexico)", "español (México)", "es-MX", "es-MX", TranslationServices.Bing | TranslationServices.Microsoft),
         ["et"] = new("Estonian", "Eesti", "et", "est"),
+        ["eu"] = new("Basque", "Euskara", "eu", "eus"),
+        ["fa"] = new("Persian", "فارسی", "fa", "fas"),
         ["fi"] = new("Finnish", "Suomi", "fi", "fin"),
+        ["fil"] = new("Filipino", "Tagalog", "fil", "fil", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["fj"] = new("Fijian", "Na Vosa Vakaviti", "fj", "fij", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["fo"] = new("Faroese", "føroyskt mál", "fo", "fao", TranslationServices.Bing | TranslationServices.Microsoft),
         ["fr"] = new("French", "Français", "fr", "fra"),
+        ["fr-CA"] = new("French (Canada)", "Français (Canada)", "fr-CA", "fr-CA", TranslationServices.Bing | TranslationServices.Microsoft),
         ["fy"] = new("Frisian", "Frysk", "fy", "fry", TranslationServices.Google),
+        ["ga"] = new("Irish", "Gaeilge", "ga", "gle"),
+        ["gd"] = new("Gaelic", "Gàidhlig", "gd", "gla", TranslationServices.Google | TranslationServices.Yandex),
```

Comment:
This language is better known as "Scottish Gaelic".

Reply:
[ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) list the primary language as Gaelic.

Reply:
Yeah but the language dictionary had the language name previously listed as "Scottish Gaelic". Changing it to "Gaelic" would break the code of people that rely on the previous name. A solution would be to add "Scottish Gaelic" as an alias.

Reply:
Google Translate list it as "Scots Gaelic".
So I changed it back to "Scottish Gaelic"

Done.